api.js

```diff
@@ -22,6 +22,17 @@ const postBotReply = (message, username) => {
   }
 };
 
+const message_parse = (query) => {
```

Comment:
Please avoid using underscores in JS. Thus, rename this to `getParsedMessage` or something.
FYI, we follow [Google's JS style guide](https://google.github.io/styleguide/javascriptguide.xml)